### DISCUSSION TITLE
Conclusions?

### DISCUSSION THREAD

**Discussion by VikingChris2010** [Lurkars/gloomhavensecretariat#332] [Q&A]
Is there a list of conclusions for the campaign sheet?  I was looking through the code for a bit but couldn't find it.  Could somebody point me to that? 

Thank you!

**Comment by Lurkars** [comment]
Sorry, I don't fully understand. Can you explain your intentions? The conclusions should be clear due to your physical scenario/section book. If data is missing, then open an issue for this. If you want to spoiler yourself somehow, look at the data/sections folder ...

**Reply by VikingChris2010** [reply]
I went through the data/sections folder and couldn't find a list.  I'm pretty sure I'm missing some but can't find a list to cross-reference.  I also wasn't sure where the conclusions were pulling from in the scenario book.  I think I accidentally deleted one that we had achieved but can't figure it out.  Thought looking at a list would be quicker than going back through the 20 scenarios wee have completed.  Thanks for the help.  This program is truly amazing and extremely helpful and time saving.  Hope you enjoyed the coffee I sent!

**Reply by Lurkars** [reply]
Okay, there is no list. In the folder are every available sections and all sections with `"conclusion": true` are conclusion for scenarios/puzzle/calendar events. If you accidentally deleted one, that's of course difficult to solve. How much time has left? You may just found it in the "undo" log and can the re-add the section.

**Reply by VikingChris2010** [reply]
Thanks!
